landice/load_compass_env.sh

```diff
@@ -0,0 +1,33 @@
+version=0.1.11
```

Comment:
Is there a need to have duplicate versions of this script for ocean and landice?

Reply:
Good call.  Up to now, there was because `landice/develop` and `ocean/develop` could have different package requirements.  But I agree that that probably no longer makes sense.  We would need to make changes to `setup_testcase.py` to look for this file in the base directory instead.  Not hard to do.  I'll do that in a separate PR and remove this.

Reply:
See #34